CONTRIBUTING.md

```diff
@@ -22,6 +22,44 @@ Runs tests and checkstyle.
 ./gradlew check
 ```
 
+## Installing/testing against a local maven repository
+
+Sometimes its helpful to build and install the bugsnag-java libraries into a
+local repository and test the entire dependency flow inside of a sample
+application.
+
+To get started:
+
+1. In the `bugsnag-java` directory, run
+   `./gradlew -Preleasing=true publishToMavenLocal`.
+   This installs `bugsnag-java` and `bugsnag-spring` into your local
+   maven repository.
+2. In your sample application `build.gradle`, add `mavenLocal()` to the *top* of
+   your `allprojects` repositories section:
+
+   ```groovy
+   allprojects {
+     repositories {
+       mavenLocal()
+       // other repos as needed
+     }
+   }
+   ```
+3. In your sample application `app/build.gradle`, add the following to the
+   dependencies section, inserting the exact version number required:
+
+   ```groovy
+   dependencies {
+     implementation 'com.bugsnag:bugsnag:[VERSION NUMBER]'
+   }
+   ```
+4. Clean your sample application and reload dependencies *every time* you
```

Comment:
I didn't find that I needed to do this, at least when testing through intellij. Is this step definitely needed?

Reply:
I think it's worth doing just to be 100% sure that we're using the local artefact. I've previously experienced the gradle dependency cache on Android not picking up changes to the build script correctly, which this step should fix.